relationships/synthesis/INFRA-HOST-to-EXT-SERVICE.yml

```diff
@@ -28,37 +28,4 @@ relationships:
         extractGuid:
           attribute: entity.guid
           entityType:
-            attribute: entity.type
-
-  - name: otelServiceHostMetric
-    version: "1"
-    origins:
-      - OpenTelemetry
-    conditions:
-      - attribute: eventType
-        anyOf: [ "Metric" ]
-      - attribute: newrelic.source
-        anyOf: [ "api.metrics.otlp" ]
-      - attribute: service.name
-        anyOf: [ "io.opentelemetry.collector" ]
```

Comment:
Maybe it makes sense to keep this rule and remove the `service.name` condition. It may also make sense to add a rule for logs. That is, an OTel service that includes a `host.id` attribute on its telemetry, will normally include it on its spans, metrics, and logs that it emits.